lib/repo/audit.js

```diff
@@ -41,20 +41,22 @@ Repo.prototype.audit = function( options ) {
 
 		// Filter out authors who have signed a CLA or CAA
 		neglectedCommits = commits.filter(function( commit ) {
-			if ( !signatures.hasOwnProperty( commit.email ) ) {
+			var email = commit.email.toLowerCase();
+			var name = commit.name.toLowerCase()
+			if ( !signatures.hasOwnProperty( email ) ) {
 				return true;
 			}
 
 			// Reject signatures with errors
-			if ( signatures[ commit.email ].errors.length ) {
+			if ( signatures[ email ].errors.length ) {
 				return true;
 			}
 
 			// At least one name must match
-			var noMatchingNames = signatures[ commit.email ].names.indexOf( commit.name ) === -1;
+			var noMatchingNames = signatures[ email ].names.indexOf( name ) === -1;
 			if ( noMatchingNames ) {
-				signatures[ commit.email ].errors.push(
-					"Name on signature (" + signatures[ commit.email ].names.join( ", " ) +
+				signatures[ email ].errors.push(
+					"Name on signature (" + signatures[ email ].names.join( ", " ) +
```

Comment:
I would expect this message to keep the original casing, even if the comparison ignores casing.